src/components/DecisioningEngine/consequenceAdapters/inAppMessageConsequenceAdapter.js

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { MESSAGE_IN_APP } from "../../Personalization/constants/schema";
+import { TEXT_HTML } from "../../Personalization/constants/contentType";
```

Comment:
We generally don't allow for imports between components. If something is shared between components, it should be in the `src/shared` modules.

BUT... this should have been caught by ESLint. There is a bug/hole in our eslint config which allows that. I've made a PR to fix it (#1050).

In the meantime, here's what should be fixed.


> @adobe/alloy@2.19.0-beta.0 lint
> eslint --cache --fix "*.js" "{src,test,scripts}/**/*.js"

```
> @adobe/alloy@2.19.0-beta.0 lint
> eslint --cache --fix "*.js" "{src,test,scripts}/**/*.js"

/Users/cmcbride/homebase/code/github.com/adobe/alloy/src/components/DecisioningEngine/consequenceAdapters/inAppMessageConsequenceAdapter.js
  12:32  error  Unexpected path "../../Personalization/constants/schema" imported in restricted zone       import/no-restricted-paths
  13:27  error  Unexpected path "../../Personalization/constants/contentType" imported in restricted zone  import/no-restricted-paths

/Users/cmcbride/homebase/code/github.com/adobe/alloy/src/components/DecisioningEngine/createEvaluableRulesetPayload.js
  12:25  error  Unable to resolve path to module '@adobe/aep-rules-engine'                            import/no-unresolved
  16:8   error  Unexpected path "../Personalization/constants/schema" imported in restricted zone     import/no-restricted-paths
  17:25  error  Unexpected path "../Personalization/constants/eventType" imported in restricted zone  import/no-restricted-paths

/Users/cmcbride/homebase/code/github.com/adobe/alloy/src/components/DecisioningEngine/createOnResponseHandler.js
  12:50  error  Unexpected path "../Personalization/constants/handle" imported in restricted zone  import/no-restricted-paths

✖ 6 problems (6 errors, 0 warnings)
```

Reply:
Addressed in the PR https://github.com/adobe/alloy/pull/1052/files